packages/contracts/foundry.toml

```diff
@@ -0,0 +1,22 @@
+[profile.default]
+src = 'contracts'
+out = 'out'
+libs = ['node_modules', 'lib']
```

Comment:
We could remove `lib` as there is no `lib` folder.
We could also remove `node_modules`, because we don't import any contracts from there.